plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-product-variations-controller.php

```diff
@@ -33,6 +33,7 @@ class WC_REST_Product_Variations_Controller extends WC_REST_Product_Variations_V
 	 * @return WP_REST_Response
 	 */
 	public function prepare_object_for_response( $object, $request ) {
+		$context  = ! empty( $request['context'] ) ? $request['context'] : 'view';
 		$data = array(
```

Comment:
This `$data` declaration does need some additional spaces so it is aligned with the `$context` one above.